coala_langserver/jsonrpc.py

```diff
@@ -78,8 +78,10 @@ def _receive(self):
         self._msg_buffer[obj.get('id') or uuid.uuid4()] = obj
 
     def read_message(self, _id=None):
-        """Read a JSON RPC message sent over the current connection. If
-        id is None, the next available message is returned."""
+        """
+        Read a JSON RPC message sent over the current connection. If
```

Comment:
this modified version is no better than the original.

revert, ignore extra rules if required